src/pages/[platform]/build-a-backend/auth/data-usage-policy/index.mdx

```diff
@@ -0,0 +1,126 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Data usage policy information',
+  description: "Review the data types gathered by the Amplify library that Apple requires you to disclose in your app's data usage policy when submitting the app to the App Store.",
```

Comment:
would recommend a shorter description

"Required Apple disclosures for your app's data usage policy when using Amplify"